cmd/kops/create_cluster.go

```diff
@@ -170,6 +172,7 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringVar(&options.Image, "image", options.Image, "Image to use")
 
 	cmd.Flags().StringVar(&options.Networking, "networking", "kubenet", "Networking mode to use.  kubenet (default), classic, external, kopeio-vxlan, weave, flannel, calico, canal.")
+	cmd.Flags().BoolVar(&options.NetworkEncrypt, "network-encrypt", options.NetworkEncrypt, "Weave network support encryption between nodes. Password stored in etcd as k8s secret weave-pass.")
```

Comment:
kms: kms was option for me. but later i got requirements support cloud in multiple cloud from multiple providers. using kms for secret will vendor lock. and encrypted network more import outside of aws - for example aws<->gce connection.
cli option: for my team main feature of kops is create cluster without complex configuration. if you will keep low amount of cli option - ansible from kargo became more preferable, disappear simplicity of kops.it is for me main feature :) click-click and k8s working - magic :) click-click--encrypt-network - k8s working and network is encrypted :)